src/main/java/uk/nhs/ctp/controllers/AuditController.java

```diff
@@ -1,40 +1,41 @@
 package uk.nhs.ctp.controllers;
 
-import java.util.List;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
 import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.data.domain.Page;
 import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.ResponseBody;
 import org.springframework.web.bind.annotation.RestController;
-import uk.nhs.ctp.entities.Audit;
-import uk.nhs.ctp.repos.AuditRepository;
-import uk.nhs.ctp.service.AuditSearchService;
-import uk.nhs.ctp.service.search.AuditSearchRequest;
-import uk.nhs.ctp.service.search.AuditSearchResultDTO;
+import uk.nhs.ctp.auditFinder.AuditFinderService;
+import uk.nhs.ctp.caseSearch.CaseSearchRequest;
+import uk.nhs.ctp.caseSearch.CaseSearchResultDTO;
+import uk.nhs.ctp.caseSearch.CaseSearchService;
 
 @CrossOrigin
 @RestController
 @RequestMapping(path = "/audit")
 @RequiredArgsConstructor
 public class AuditController {
 
-	private final AuditRepository auditRepository;
-	private final AuditSearchService auditSearchService;
+	@Qualifier("enhanced")
+	private final ObjectMapper mapper;
+	private final AuditFinderService auditFinder;
+	private final CaseSearchService caseSearchService;
 
 	@GetMapping(path = "/{id}")
-	public @ResponseBody
-	List<Audit> getAudit(@PathVariable Long id) {
-		return auditRepository.findAllByCaseId(id);
+	public String getAudit(@PathVariable Long id) throws IOException {
```

Comment:
the UI seems to stringify this anyway? Guessing that's why we now just return a string here? Can the UI be updated to expect the string rather than the objects?

Reply:
Looking at it now, the UI (`audit-display.component.ts`) receives the audit list as a JSON array and then stringifies it, it keeps both versions but never uses the stringified one :/
I'll remove the unnecessary stringify